Userland/Libraries/LibWeb/DOM/Document.cpp

```diff
@@ -920,4 +920,14 @@ void Document::set_cookie(String cookie_string, Cookie::Source source)
         page->client().page_did_set_cookie(m_url, cookie.value(), source);
 }
 
+String Document::dump_dom_tree()
```

Comment:
Maybe this should specifiy that it's JSON, so `dump_dom_tree_as_json()` or something like that.
And can't this function be `const`? (I don't know, I'm just asking.)